javascript-light-bulb/main.js

```diff
@@ -1,15 +1,16 @@
 var $button = document.querySelector('button');
+var $class = $button.className;
```

Comment:
This is still the same problem, you are still making decisions based on the current classes of the element in the DOM. I would suggest using a boolean to keep track of if the light is currently on or off, and re-assign the value to the opposite boolean every time the user clicks the light switch. You could name the variable something like `isLightOn`, and its starting value should be whatever value matches how the page loads. So for example, if the document loads with the light off, then the variable would start as `var isLightOn = false;`